ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -134,7 +134,7 @@ class TabLocationView: UIView {
     secureContentStateButton.isHidden = false
 
     // Consider brave translate
-    translateButton.isHidden = translationState == .unavailable
+    translateButton.isHidden = true
```

Comment:
I dont think this is necessary since `translateButton` should be a subview in `leadingItemStackView` which sets all its views to hidden at the start of the method

Reply:
Fixed